cypress/integration/registration_landing_page.spec.js

```diff
@@ -0,0 +1,39 @@
+import { dataTestId } from '../../src/utils/dataTestIds';
+
+const {
+  registrationLandingPage: { projectTitle, status },
+  projectLandingPage: { generalInfoBox, participantsAccordion, resultsAccordion, scientificSummaryAccordion },
+} = dataTestId;
+
+describe('User opens Landing Page for Registration', () => {
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  it('The User should be able to open Landing Page from My Registrations', () => {
+    cy.mocklogin();
+    cy.get('[data-testid=my-registrations]').click({ force: true });
+    cy.get('[data-testid^=open-registration]').eq(0).click({ force: true });
+
+    cy.url().should('include', '/public');
+    cy.get(`[data-testid=${status}]`).should('be.visible');
+  });
+
+  it('Anonymous user should be able to open Landing Page for Registration', () => {
+    cy.visit('/registration/123/public');
+    cy.get('[data-testid=my-registrations]').should('not.exist');
+    cy.get(`[data-testid=${status}]`).should('not.exist');
+  });
+
+  it('Project should have a link to Landing Page for Project', () => {
+    cy.visit('/registration/123/public');
+    cy.get(`[data-testid=${projectTitle}]`).should('be.visible');
```

Comment:
ser dere bruker should be visible og not.exist.  Er det ikke mer logisk å bruke "should exist". i praksis spiller det vel liten rolle, men jeg tenker at om man sjekker om noe er synlig, så er det fordi det har en mulighet til å være hidden, eller bli presset ut av skjermen (om det ikke er det man tester). Man trenger vel egentlig ingen av delene, men det er kjekt for lesbarheten.

Reply:
Enig med @pcb-unit

Reply:
Jeg tenker også at hvis scenarioet sier noe som `user should see...` så bør man bruke `should('be.visible')` mens `should have` så bruker man `should('exist')`

Reply:
`should('exist')` er egentlig redundant, men jeg syns det gjør det lettere å lese testen så foretrekker å bruke den i stedet for bare en `get`

Reply:
En siste nyanse på bruk av `should('exist')`, hvis du bruker negative assertions (f.eks `should('not.be.visible')`) så vil ikke den implisitte `should('exist')` slå til på en `get`.

F.eks:
`cy.get('data-testid=something').should('not.be.visible')` vil ikke feile om elementet med `data-testid=something` ikke eksisterer i siden. Da er det bedre å bruke
`cy.get('data-testid=something').should('exist).should('not.be.visible')`

Reply:
👍 
endrer til `should('exist')`